### PR TITLE
Update new logger for the files under package "service"

### PR DESCRIPTION
**Pull request by redkafei**

### CODE REVIEW

File: service/utils/authutils.go [hunk 1]
```diff
@@ -71,19 +71,19 @@ func FilterAccess(username string, authenticated bool, a *token.ResourceActions)
...
-						log.Printf("Error occurred in CheckExistProject: %v", err)
+						log.Errorf("Error occurred in CheckExistProject: %v", err)
 						return
 					}
 					if exist {
 						permission = "RW"
 					} else {
 						permission = ""
-						log.Printf("project %s does not exist, set empty permission for admin", projectName)
+						log.Infof("project %s does not exist, set empty permission for admin", projectName)
```
Comment (reasonerjt) [inline service/utils/authutils.go]: You probably need to add \n at the end of the string

File: service/notification.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/vmware/harbor/dao"
 	"github.com/vmware/harbor/models"
 	svc_utils "github.com/vmware/harbor/service/utils"
+	log "github.com/vmware/harbor/utils/log"
```
Comment (reasonerjt) [inline service/notification.go]: I think "github.com/vmware/harbor/utils/log" should be ok you don't need to rename it as log.

File: service/token.go [hunk 3]
```diff
@@ -38,14 +38,14 @@ type TokenHandler struct {
 func (a *TokenHandler) Get() {
 
 	request := a.Ctx.Request
-	log.Println("request url: " + request.URL.String())
+	log.Info("request url: " + request.URL.String())
```
Comment (reasonerjt) [inline service/token.go]: Could you use Infof?

File: service/utils/registryutils.go [hunk 4]
```diff
@@ -34,7 +35,7 @@ func BuildRegistryURL(segments ...string) string {
 	url := registryURL + "/v2"
 	for _, s := range segments {
 		if s == "v2" {
-			log.Printf("unnecessary v2 in %v", segments)
+			log.Infof("unnecessary v2 in %v", segments)
```
Comment (reasonerjt) [inline service/utils/registryutils.go]: This should be a debug message

File: service/utils/registryutils.go [hunk 5]
```diff
@@ -59,7 +60,7 @@ func RegistryAPIGet(url, username string) ([]byte, error) {
 	} else if response.StatusCode == http.StatusUnauthorized {
 		authenticate := response.Header.Get("WWW-Authenticate")
 		str := strings.Split(authenticate, " ")[1]
-		log.Println("url: " + url)
+		log.Info("url: " + url)
```
Comment (reasonerjt) [inline service/utils/registryutils.go]: This should be a debug message, try using Debugf

File: service/notification.go (hunk 2 above)
Comment (redkafei) [inline service/notification.go]: fixed

File: service/token.go (hunk 3 above)
Comment (redkafei) [inline service/token.go]: fixed

File: service/utils/registryutils.go (hunk 4 above)
Comment (redkafei) [inline service/utils/registryutils.go]: fixed

File: service/utils/registryutils.go (hunk 5 above)
Comment (redkafei) [inline service/utils/registryutils.go]: fixed

File: service/utils/registryutils.go [hunk 6]
```diff
@@ -59,7 +60,7 @@ func RegistryAPIGet(url, username string) ([]byte, error) {
 	} else if response.StatusCode == http.StatusUnauthorized {
 		authenticate := response.Header.Get("WWW-Authenticate")
 		str := strings.Split(authenticate, " ")[1]
-		log.Println("url: " + url)
+		log.Debugf("url: " + url)
```
Comment (reasonerjt) [inline service/utils/registryutils.go]: let's use placeholder, i.e "url: %s"

**Comment by xiaods** [comment]
LGTM

File: service/utils/registryutils.go (hunk 6 above)
Comment (redkafei) [inline service/utils/registryutils.go]: Just update as your suggestion

**Pull request merged**